lms/resources/_js_config/__init__.py

```diff
@@ -215,7 +215,7 @@ def enable_lti_launch_mode(self, course, assignment: Assignment):
             },
         }
 
-    def enable_file_picker_mode(self, form_action, form_fields):
+    def enable_file_picker_mode(self, form_action, form_fields, prompt_for_title=False):
```

Comment:
We could use the plugin here.

I reckon this is better, this method is told to use or not the promt_for_title and it doesn't care why.